### PR TITLE
Fix the imports based on new bullet-core

### PR DESCRIPTION
**Pull request by hbxie**
I'll hold the pom.xml until bullet-core is released.

### CODE REVIEW

**Comment by akshaisarma** [comment]
@hbxie Thanks. We need to wait till bullet-core-0.3.0 is released first though. That's why the build is failing. I will wait till the Storm and Spark implementations of 0.3.0 are tested first before releasing 0.3.0.

**Comment by hbxie** [comment]
Yes. I'll hold it until bullet-core is released.

**Comment by akshaisarma** [comment]
Can you bump the major version to 0.3.0 as well?

File: pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <bullet.core.version>0.2.5</bullet.core.version>
+        <bullet.core.version>0.3.0</bullet.core.version>
```
Comment (akshaisarma) [inline pom.xml]: Can we also bump this project from 0.2.1-SNAPSHOT to 0.3.0-SNAPSHOT since this breaks interface

Comment (hbxie) [reply pom.xml]: Done.

**Pull request merged**